src/Goteo/Model/ImpactData.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/*
+ * This file is part of the Goteo Package.
+ *
+ * (c) Platoniq y Fundación Goteo <fundacion@goteo.org>
+ *
+ * For the full copyright and license information, please view the README.md
+ * and LICENSE files that was distributed with this source code.
+ */
+
+namespace Goteo\Model;
+
+use Goteo\Core\Model;
+use Goteo\Model\Image;
+use Goteo\Library\Text;
+
+use Goteo\Application\Exception\ModelNotFoundException;
+
+class ImpactData extends Model {
+
+	public
+		$id,
+		$title,
+		$data,
+        $data_unit,
+		$description,
+		$image,
+		$lang;
+
+    protected $Table = 'impact_data';
+	static protected $Table_static = 'impact_data';
+
+    public static function getLangFields() {
+        return ['title', 'data', 'data_unit', 'description'];
+    }
+
+    public static function get($id) {
+
+        list($fields, $joins) = self::getLangsSQLJoins($lang);
+
+        $sql = "SELECT
+                    id,
+                    $fields,
+                    image,
+                    lang
+                FROM impact_data
+                $joins
+                WHERE id = :id";
```

Comment:
For the next time that we need to create a new model, I would strongly suggest that we take the time to configure Doctrine ORM and we start using it for that model.